config/v1.6/aws-k8s-cni.yaml

```diff
@@ -99,7 +99,7 @@ spec:
       tolerations:
         - operator: Exists
       containers:
-        - image: 602401143452.dkr.ecr.us-west-2.amazonaws.com/amazon-k8s-cni:v1.6.3
+        - image: 602401143452.dkr.ecr.us-west-2.amazonaws.com/amazon-k8s-cni:v1.6.4-rc1
```

Comment:
Should we add cii-metrics helper rc1?

Reply:
For `v1.6.4` final we should, but since no metrics have changed in this release candidate, `v1.6.3` works for testing. Same with the other configs, we will update once they are built and published.